### PR TITLE
Receiving invalid syntax error when using MySql with RDBMS under inte…

### PR DESCRIPTION
**Pull request by quadoss**
…rperter

Internal Jira: CDPD-45759

## What changes were proposed in this pull request?
Syntax change i.e. adding braces

## How was this patch tested?
In customer env.

### CODE REVIEW

**Review by Akhilsnaik — DISMISSED** [review]
The Same Need to be corrected here too : https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/connectors/jdbc.py#L52

**Comment by quadoss** [comment]
> The Same Need to be corrected here too : https://github.com/cloudera/hue/blob/master/desktop/libs/notebook/src/notebook/connectors/jdbc.py#L52

Thanks @Akhilsnaik for the review.  Let me check where all the changes are needed and will push them

File: desktop/libs/notebook/src/notebook/connectors/rdbms.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def decorator(*args, **kwargs):
       if 'Invalid query handle' in message or 'Invalid OperationHandle' in message:
         raise QueryExpired(e)
       else:
-        raise QueryError, message, sys.exc_info()[2]
+        raise(QueryError, message, sys.exc_info()[2])
```
[the remark this answers is not in the document]
Comment (Harshg999) [reply desktop/libs/notebook/src/notebook/connectors/rdbms.py:43]: Reference: https://docs.python.org/3.8/library/exceptions.html#BaseException.with_traceback

[the remark this answers is not in the document]
Comment (quadoss) [reply desktop/libs/notebook/src/notebook/connectors/rdbms.py]: Thanks @Harshg999.  Made the needed changes.

File: desktop/libs/notebook/src/notebook/connectors/jdbc.py [hunk 2]
```diff
@@ -45,7 +45,10 @@ def decorator(*args, **kwargs):
...
       message = force_unicode(smart_str(e))
       if 'error occurred while trying to connect to the Java server' in message:
-        raise QueryError, _('%s: is the DB Proxy server running?') % message, sys.exc_info()[2]
+        if sys.version_info[0] > 2:
+          raise QueryError(_('%s: is the DB Proxy server running?') % message).with_traceback(sys.exc_info()[2])
+        else:
+          raise QueryError, _('%s: is the DB Proxy server running?') % message, sys.exc_info()[2]
       elif 'Access denied' in message:
         raise AuthenticationRequired, '', sys.exc_info()[2]
       else:
```
Comment (Akhilsnaik) [inline desktop/libs/notebook/src/notebook/connectors/jdbc.py:54]: the same change need to be added here too : https://github.com/cloudera/hue/blob/64476fb530e6d16fa8869c4ec144f807095140e3/desktop/libs/notebook/src/notebook/connectors/jdbc.py#L55

```
     else:
        raise QueryError, message, sys.exc_info()[2]
```

**Review by Harshg999 — APPROVED** [review]
Looks good! Please do test it in the cluster.

**Pull request merged**